ownership-chain/runtime/src/tests.rs

```diff
@@ -1,11 +1,41 @@
 use super::*;
-use frame_support::assert_ok;
+use fp_rpc::runtime_decl_for_ethereum_runtime_rpc_api::EthereumRuntimeRPCApiV5;
+use frame_support::{
+	assert_ok,
+	traits::tokens::{fungible::Balanced, Precision},
+};
 use sp_core::U256;
+use std::str::FromStr;
+
+const ALICE: &str = "0xf24FF3a9CF04c71Dbc94D0b566f7A27B94566cac";
+
+// Build genesis storage according to the mock runtime.
+pub fn new_test_ext() -> sp_io::TestExternalities {
+	frame_system::GenesisConfig::<Runtime>::default()
+		.build_storage()
+		.unwrap()
+		.into()
+}
 
 #[test]
-fn test_block_and_gas_limit_constants() {
-	use crate::Runtime;
+fn minimum_balance_should_be_0() {
+	assert_eq!(Balances::minimum_balance(), 0);
+}
+
+#[test]
+fn deposit_smallest_unit_should_succeed() {
```

Comment:
I think with `send_1_wei_to_wallet_with_0_balance_should_increase_balance_by_1_wei`, this test becomes unnecessary, since all the actions/checks that are done here is done in `send_1_wei_to_wallet_with_0_balance_should_increase_balance_by_1_wei` as well

Reply:
sgtm